### DISCUSSION TITLE
Is there a way to increase the width of the whole shell menu?

### DISCUSSION THREAD

**Discussion by milindgoel15** [moudey/Shell#153] [Q&A]
![image](https://user-images.githubusercontent.com/45682747/216567233-0fc1769b-58e2-4418-bf49-ee0ba4605f0e.png)
basically increasing the main layout width.

**Comment by moudey** [comment]
Open `shell.nss`, add this to `theme` section
```
item {
  padding.left=50
  padding.right=50
}
```
save, press `ctrl` + `right click` to reload settings.

**Reply by milindgoel15** [reply]
ah, i was doing it wrong hmm. I added the padding params outside the item block.

For some strange reason, after I installed shell, the app opening became a lil slower than before.

And after I uninstalled the shell, the context menu wasn't restored to the one offered by win11 but the win10 version was restored.

**Reply by moudey** [reply]
The slowness may be caused by the effect of acrylic.

**Reply by milindgoel15** [reply]
You can mark this discussion as answered.
